Algorithms/Divide-and-Conquer/Strassens Algorithm.cpp

```diff
@@ -0,0 +1,129 @@
+#include <bits/stdc++.h>
+#include <vector>
```

Comment:
No need of additional header files, if you have already included "bits/stdc++.h"